src/Packagist/WebBundle/Resources/config/services.yml

```diff
@@ -26,14 +26,12 @@ services:
 
     packagist.user_repository:
         class: Packagist\WebBundle\Entity\UserRepository
-        factory_service: doctrine
-        factory_method: getRepository
+        factory: [@doctrine, getRepository]
```

Comment:
No, the `@` is a reserved character now (because -reasons-, don't ask me, I really don't know). So you should always quote the identifiers that use it :)